### PR TITLE
Test file name modification

### PR DESCRIPTION
**Pull request by thisaltennakoon**

### CODE REVIEW

**Review by tharikaGitHub — APPROVED** [review]

File: tests/cypress/integration/devportal/000-general/03-self-signup.spec.js [hunk 1]
```diff
@@ -21,10 +21,15 @@ function getSuperTenantEmail(username) {
     return `${username}@test.com`;
 }
 
+function getTenantUser(username, domain) {
```
Comment (madushadhanushka) [inline tests/cypress/integration/devportal/000-general/03-self-signup.spec.js:24]: Shall we move this code into a util class?

Comment (thisaltennakoon) [reply tests/cypress/integration/devportal/000-general/03-self-signup.spec.js]: Moved in https://github.com/wso2/apim-apps/pull/375/commits/01d02797e5a18f82c29825c51b653442ed99d964

**Review by madushadhanushka — APPROVED** [review]

File: tests/cypress/integration/devportal/000-general/03-self-signup.spec.js [hunk 2]
```diff
@@ -215,5 +216,4 @@ describe("Self SignUp", () => {
         cy.updateTenantConfig(tenantAdminUsername, tenantAdminPassword, testTenant, tenantConfigJson);
         cy.enableSelfSignUpInCarbonPortal(tenantAdminUsername, tenantAdminPassword, testTenant);
     })
-
 });
```
Comment (chameerar) [inline tests/cypress/integration/devportal/000-general/03-self-signup.spec.js:219]: Please add the missing new line

Comment (thisaltennakoon) [reply tests/cypress/integration/devportal/000-general/03-self-signup.spec.js:219]: Fixed in https://github.com/wso2/apim-apps/pull/454

**Pull request merged**